dev/io.openliberty.jakarta.validation.v31_fat/test-applications/val31/src/val31/web/Registeration.java

```diff
@@ -0,0 +1,18 @@
+/**
+ *
+ */
+package val31.web;
+
+import jakarta.validation.constraints.AssertTrue;
+import jakarta.validation.constraints.NotNull;
+
+/**
+*
+*/
+public record Registeration(
```

Comment:
The record name is spelled incorrectly, it should be `Registration`

Reply:
updated